src/Web/Router/Route.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Web\Router;
+
+use App\Web\Request\Request;
+use App\Web\Response\Response;
+
+class Route
+{
+    private array $handlers = [];
+
+    public function __construct(
+        private readonly string $pattern,
+        private readonly Response $notAllowed,
```

Comment:
What is the reason of this to be a dependency? 

This seems like a default value that is being returned by handle method so it could be a) passed as argument to method or b) just simply generated inside method